bin/ch/WScriptJsrt.cpp

```diff
@@ -1073,6 +1074,115 @@ JsValueRef __stdcall WScriptJsrt::LoadTextFileCallback(JsValueRef callee, bool i
     return returnValue;
 }
 
+int js_fgets(char* buf, int size, FILE* file)
+{
+	int  i, n, c;
+	bool crflag;

```

Comment:
Can you initialize these or move the declaration to the point of initialization? This function doesn't have gotos, so there's not as much the desire to avoid jumping past declarations that there is with other code.

Reply:
good idea.